### ISSUE TITLE
Build failing: ...index.ios.js tries to require 'react-native', but there are several files providing this module...

### ISSUE THREAD

**Issue by spencewine** [tipsi/tipsi-stripe#321]
I'm getting a dependency collision with react-native when following the installation steps:

![screen shot 2018-06-27 at 10 55 47 am](https://user-images.githubusercontent.com/1612742/41985916-3ff74a3e-79fa-11e8-8d8e-eadfc36af3f0.png)

Steps I took:

`npm install --save tipsi-stripe`

opened pod file in ios directory, added:
`pod 'Stripe', '~> 11.2.0'`
ran `pod install`
added `node_modules/tipsi-stripe/ios/TPSStripe/` to project
ran `react-native link tipsi-stripe` but that didn't work so manually linked the project

here is my podfile

```
# Uncomment the next line to define a global platform for your project
 platform :ios, '9.0'


target 'App' do
  # Uncomment the next line if you're using Swift or would like to use dynamic frameworks
  # use_frameworks!

  # Pods for LGSAppRedux
    pod 'Google/SignIn'

    pod 'Stripe', '~> 11.2.0'

  pod 'RNVectorIcons', :path => '../node_modules/react-native-vector-icons'

  pod 'react-native-orientation', :path => '../node_modules/react-native-orientation'

  pod 'RNAWSCognito', :path => '../node_modules/amazon-cognito-identity-js'

  # pod 'tipsi-stripe', :path => '../node_modules/tipsi-stripe'

  target 'App' do
    inherit! :search_paths
    # Pods for testing
  end

end
```
header search paths in the TPSStripe proj: 
![screen shot 2018-06-27 at 11 07 52 am](https://user-images.githubusercontent.com/1612742/41985979-6a301826-79fa-11e8-85eb-cb56abf2bb19.png)

search paths in my project:
![screen shot 2018-06-27 at 11 08 42 am](https://user-images.githubusercontent.com/1612742/41986006-7f666506-79fa-11e8-8387-eb05f05cc84e.png)

**Comment by isnifer** [comment]
@spencewine Hi! I see you tried to install `tipsi-stripe` via podspec.
So, how did you install it? 
Try to remove it from Podfile completely, clean `build` folder (`DerivedData` folder in your Xcode settings or maybe you defined it manually) and build again.

**Comment by spencewine** [comment]
@isnifer I've removed it from the pod file, installed and still getting the same error. `tipsi-stripe` was added to the file automatically when I ran `react-native link tipsi-stripe`, I did not add it manually

**Comment by spencewine** [comment]
@isnifer okay I figured out the source of that error message. It wasn't due to tipsi-stripe, but due to the other pods I had in the pod file. Those libraries were already linked and did not need to be installed via cocoa-pods. They hadn't been a problem before hence why I thought the installation of tipsi-stripe was causing the problem, but I'm guessing no one ran pod install on the ios file until I did when adding the Stripe pod. Thanks for your help, your comment made me realize it could be the other pods.
